pages/housing-assistance/index.md

```diff
@@ -174,7 +174,7 @@ VA housing assistance can help Servicemembers, Veterans, and their surviving spo
 <div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
     <h4 class="usa-alert-heading">How do I get help if I'm homeless or at risk of becoming homeless?</h4>
-		<div class="form-expanding-group borderless-alert additional-info-container" data-event="nav-crisis-homelessness-expander">
+		<div id="crisis-line-details" class="form-expanding-group borderless-alert additional-info-container" data-event="nav-crisis-homelessness-expander">
```

Comment:
It would be cool to consolidate the id and the `data-event` into one attribute at some point.